fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnService.java

```diff
@@ -37,6 +37,7 @@ public void downloadFiles() {
             Audit.info(String.format("%d new messages in FIKS", forsendelses.size()));
         }
         for(Forsendelse forsendelse : forsendelses) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS135)